security-center/snippets/system-test/v2/notifications.test.js

```diff
@@ -41,7 +41,7 @@ describe('Client with Notifications v2', async () => {
 
   before(async () => {
     const configId = 'notif-config-test-node-create-' + uuidv1();
-    topicName = 'test_topic';
+    topicName = 'notifications-sample-topic';
```

Comment:
praise: I really like that we're changing the name to be more specific.

suggestion: Prefix with the product as well, to make it clearly differentiated from notification functionality in other products. Don't add the name "sample" because that's context for all resources.

```suggestion
    topicName = 'securitycenter-notifications-topic';
```